src/app/components/challenge/challengephases/phasecard/phasecard.component.ts

```diff
@@ -81,10 +81,17 @@ export class PhasecardComponent implements OnInit {
     SELF.apiCall = (params) => {
       const FORM_DATA: FormData = new FormData();
       for (const key in params) {
-        if (params[key]) {
-          FORM_DATA.append(key, params[key]);
+       if (this.challengeService.isValidSubmission(params)) {
+          if (params[key]) {
+              FORM_DATA.append(key, params[key]);
+            }
+          } else {
+            SELF.globalService.showToast('error', 'Invalid maximum submissions value!!');
```

Comment:
Minor: Can we please remove that extra exclamation (`!`) from here.

And also I think instead of showing it on toast we should show it on the modal itself cause it looks weird that it closes the modal and also remove all the entered value. WDYT @Kajol-Kumari

Reply:
@Sanji515 we are showing most of the errors on toast only and also in editing phase detail, we are just applying check exclusively on the submission count only. So, i think it makes sense to show the error on toast and abort the editing when entered an unacceptable value. Also it keeps all the logic in the controller itself this way. What you say?

Reply:
As it is just a validation error so I think better not to close the modal and start all over again instead we should show that validation message there only like we show the validation message for phase name when we clear the text from the input.

Reply:
@Sanji515 I shifted the validation message on the modal instead of showing it on toast. Please check once and if any changes are required, please let me know.